### PR TITLE
API-1862: Update sync guide with new asset and record code filter

### PR DESCRIPTION
**Pull request by LevFlavien**
https://akeneo.atlassian.net/browse/API-1862

### CODE REVIEW

**Review by LevFlavien — COMMENTED** [review]
I think example JSONs should use the same data as snippets (eg `kartell` <> `record_code_1`)  
If applied, this needs further review across the whole guide

File: content/getting-started/synchronize-pim-products-6x/step-6.md [hunk 1]
```diff
@@ -137,13 +137,13 @@ As for reference entities, some asset family attributes may be simple select or
 
 ### 3 - Collect assets
 
-Simply call [the following API endpoint](https://api.akeneo.com/api-reference.html#Asset).
+Call the Assets [API endpoint](https://api.akeneo.com/api-reference.html#Asset) and filter by the Asset codes you previously fetched.
```
Comment (tomglvng) [inline content/getting-started/synchronize-pim-products-6x/step-6.md:140]:
```suggestion
Call the assets [API endpoint](https://api.akeneo.com/api-reference.html#Asset) and filter by the asset codes you previously fetched.
```

File: content/getting-started/synchronize-pim-products-6x/step-5.md [hunk 2]
```diff
@@ -87,13 +87,13 @@ As for products, some attributes may be simple select or multi select type (`pim
 
 ### 3 - Collect reference entity records
 
-Again, an [API call](https://api.akeneo.com/api-reference.html#Referenceentityrecord) will save your life.
+Call the Records [API endpoint](https://api.akeneo.com/api-reference.html#Referenceentityrecord) and filter by the Record codes you previously fetched.
```
Comment (tomglvng) [inline content/getting-started/synchronize-pim-products-6x/step-5.md:90]:
```suggestion
Call the records [API endpoint](https://api.akeneo.com/api-reference.html#Referenceentityrecord) and filter by the record codes you previously fetched.
```

**Comment by tomglvng** [comment]
> I think example JSONs should use the same data as snippets (eg `kartell` <> `record_code_1`) If applied, this needs further review across the whole guide

I agree, it would be more clear. I had a look, and changes have to be done in `Synchronize References entities` and `Synchornize Assets` only.

Can you update it in this PR, please?

**Comment by tomglvng** [comment]
Ticket has been closed

**Pull request closed without merging**